mne/annotations.py

```diff
@@ -810,64 +844,31 @@ def __init__(
         duration,
         description,
         hed_strings,
-        hed_version="8.3.0",  # TODO @VisLab what is a sensible default here?
+        hed_version="8.3.0",
```

Comment:
This is the right version, but we will be releasing 8.4.0 soon. It is backwards compatible as far as annotations, but has internal support for linking terms to other ontologies.  It will be a seamless change as far as MNE.